src/main/java/org/opentripplanner/model/transfer/StopTransferPoint.java

```diff
@@ -11,35 +9,29 @@
 
   private final StopLocation stop;
 
+
   public StopTransferPoint(StopLocation stop) {
     this.stop = stop;
   }
 
-  @Override
   public StopLocation getStop() {
     return stop;
   }
 
   @Override
-  public int getSpecificityRanking() {
-    return 0;
+  public boolean applyToAllTrips() {
```

Comment:
Shouldn't this be called `appliesToAllTrips` since it's asking a question rather than doing something?

Reply:
I renamed this field.